libbindgen/build.rs

```diff
@@ -38,13 +38,14 @@ mod testgen {
                         .replace(|c| !char::is_alphanumeric(c), "_")
                         .replace("__", "_")
                         .to_lowercase();
-                    let _ = writeln!(dst, "test_header!(header_{}, {:?});",
-                        func, entry.path());
+                    writeln!(dst, "test_header!(header_{}, {:?});",
+                             func, entry.path()).unwrap();
```

Comment:
I didn't because `cargo fmt` doesn't detect the build.rs file apparently. Probably worth making it recognise it as a followup.